src/org/zaproxy/zap/extension/ascanrules/TestCrossSiteScriptV2.java

```diff
@@ -332,10 +334,6 @@ public void scan(HttpMessage msg, String param, String value) {
 									"", contexts2.get(0).getTarget(), contexts2.get(0).getMsg());
 							attackWorked = true;
 						}
-						if (!attackWorked) {
-							log.debug("Failed to find vuln in with simple onmounseover "

```

Comment:
This is executed (the unit tests cover this line)?

Reply:
Confirmed:
![image](https://user-images.githubusercontent.com/7570458/37340934-2004fef4-2696-11e8-8b14-18b90af3006d.png)

Reply:
I am sorry, I didn't think of a case where context2 is not null and empty

Reply:
Nonprobkem, thanks for looking into it.